cloud-gateway-service/src/main/resources/application.yml

```diff
@@ -7,6 +7,8 @@ eureka:
         metadata-map:
             apiml:
                 service.apimlId :  ${apiml.service.apimlId}
+                apiml.authentication.scheme: x509
```

Comment:
Cloud gateway is now onboarded with x509? What is the reason?

Reply:
Actually, it is about Gateways - new and old ones. For SCGW it is necessary since v3 but the behaviour is the same. Communication between GW has to be with x509 to transport information about the certs. Basically, this schema is used only by GW and set for GW - it means SCGW > GW. The communication should be like bypass + client cert. And it is for now the schema x509. Even I guess the x509 schema would be improved, it is now once possibility because the fix removes signing by client cert as default...